examples/node-echo/README.md

```diff
@@ -0,0 +1,54 @@
+Node.js `echo` Service on OpenShift!
+-----------------
+
+This example will serve an http response of various "machine" stats from the "machine" the Node.js app is running on to [http://host:8080](http://host:8080).
+
+### OpenShift setup ###
+
+I just used the docker all-in-one launch as described in the [origins project](https://github.com/openshift/origins).
+
+### The project ###
+
+If you don't have a project setup all ready, go ahead and take care of that
+
+        $ openshift ex new-project node-echo --display-name="node-echo" --description="Sample Node.js app" --admin=username
+
+That's it, project has been created.  Though it would probably be good to set your current project to this, such as:
+
+        $ osc project node-echo
```

Comment:
Well, I did the demo before his change and now `osc new-project` doesn't accept --admin flag.  So I need to update to whatever it should be now.  It still works with `openshift ex new-project` as documented.  I plan to update once I work out the new route.